src/main/resources/service_config.json

```diff
@@ -90,6 +90,17 @@
         "CREATE",
         "READ"
       ]
+    },
+    {
+      "id": "claim-store-api",
+      "caseTypeId": ["*"],
+      "jurisdictionId": "*",
```

Comment:
Can you specify your jurisdiction and caseTypes please. Also do you use all 4 apis? or just upload  and download only (READ, CREATED)?

Reply:
Jurisdiction is CMC, I have mentioned. CaseType id can be different based on money claims and legal rep claims, so kept it a *.
You are right, only upload and download. so removed the rest of two.